documentation/src/main/markdown/runningdetect/runincontainer.md

```diff
@@ -18,7 +18,7 @@ To run a container built from a [detect_product_short] image, use the Docker CLI
 
 Find available images [here](https://hub.docker.com/repository/docker/blackducksoftware/detect).
 
-Find the source for them (Dockerfiles) [here](https://github.com/blackducksoftware/synopsys-detect-docker).
+Find the source for them (Dockerfiles) [here](https://github.com/jenkinsci/detect-plugin).
```

Comment:
@sig-cpotts This is possibly a typo! The source for docker files will most likely change to https://github.com/blackducksoftware/detect-docker, and not the plugin one